.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 out/
+dist/
```

Comment:
I notice its now building to dist folder can we keep it to build to out as per original build config